diffpy/pdfmorph/pdfmorphapp.py

```diff
@@ -227,9 +239,130 @@ def custom_error(self, msg):
+        # Sort by temperature
+        try:
+            target_list = tools.temperature_sort(target_list)
+        except ValueError:
+            parser.custom_error("All file names in directory must end in _#K.gr or _#K.cgr (where # is a float) to use "
+                                "the --temperature option.")
+    else:
+        # Default is alphabetical sort
+        target_list.sort()
+
+    # Disable single morph plotting and saving
+    plot_opt = opts.plot
+    opts.plot = False
+    save_opt = opts.savefile
+    opts.savefile = None
+
+    # Do not morph morph_file against itself if it is in the same directory
+    if morph_file in target_list:
+        target_list.remove(morph_file)
+
+    # Morph morph_file against all other files in target_directory
+    results = []
+    for target_file in target_list:
+        if target_file.is_file:
+            results.append([
+                target_file.name,
+                single_morph(parser, opts, [morph_file, target_file], stdout_flag=False),
+            ])
+
+    # Input parameters used for every morph
+    inputs = [None, None]
+    inputs[0] = f"# Morphed file: {morph_file.name}"
+    inputs[1] = "\n# Input morphing parameters:"
+    inputs[1] += f"\n# scale = {opts.scale}"
+    inputs[1] += f"\n# stretch = {opts.stretch}"
+    inputs[1] += f"\n# smear = {opts.smear}"
+
+    # If print enabled
+    if stdout_flag:
+        # Separated for saving
+        print(f"\n{inputs[0]}{inputs[1]}")
+
+        # Results from each morph
+        for entry in results:
+            outputs = f"\n# Target: {entry[0]}\n"
+            outputs += "# Optimized morphing parameters:\n"
+            outputs += "\n".join(f"# {i[0]} = {i[1]:.6f}" for i in entry[1])
+            print(outputs)
+
+    rws = []
+    target_labels = []
+    results_length = len(results)
+    for entry in results:
+        if opts.temp:
+            name = entry[0]
+            target_labels.append(tools.extract_temperatures([name])[0])
+        else:
+            target_labels.append(entry[0])
+        for item in entry[1]:
+            if item[0] == "Rw":
+                rws.append(item[1])
 
-    if len(pargs) != 2:
-        parser.error("You must supply FILE1 and FILE2")
+    if save_opt is not None:
+        # Save table of Rw values
+        try:
+            with open(save_opt, 'w') as outfile:
+                # Header
+                print(f"{inputs[0]}\n{inputs[1]}", file=outfile)
+                if opts.temp:
+                    print(f"\n# L T(K) Rw", file=outfile)
+                else:
+                    print(f"\n# L Target Rw", file=outfile)
+
+                # Table
+                for idx in range(results_length):
+                    print(f"{target_labels[idx]} {rws[idx]}", file=outfile)
+                outfile.close()
+
+                # Output to stdout
+                path_name = Path(outfile.name).absolute()
+                save_message = f"\n# Rw table saved to {path_name}"
```

Comment:
maybe don't prepend a blank line?   It rarely looks good.

Reply:
Here is a sample output.

```# Morphed file: SrFe2As2_150K.gr
# Input morphing parameters:
# scale = None
# stretch = None
# smear = None

# Target: SrFe2As2_174K.gr
# Optimized morphing parameters:
# rmax = 100.010000
# rmin = 0.000000
# rstep = 0.010000
# Rw = 0.072189
# Pearson = 0.997625

# Target: SrFe2As2_180K.gr
# Optimized morphing parameters:
# rmax = 100.010000
# rmin = 0.000000
# rstep = 0.010000
# Rw = 0.085544
# Pearson = 0.996714

# Target: SrFe2As2_186K.gr
# Optimized morphing parameters:
# rmax = 100.010000
# rmin = 0.000000
# rstep = 0.010000
# Rw = 0.098680
# Pearson = 0.995607

# Target: SrFe2As2_192K.gr
# Optimized morphing parameters:
# rmax = 100.010000
# rmin = 0.000000
# rstep = 0.010000
# Rw = 0.117966
# Pearson = 0.993501

# Target: SrFe2As2_198K.gr
# Optimized morphing parameters:
# rmax = 100.010000
# rmin = 0.000000
# rstep = 0.010000
# Rw = 0.150522
# Pearson = 0.989150

# Target: SrFe2As2_204K.gr
# Optimized morphing parameters:
# rmax = 100.010000
# rmin = 0.000000
# rstep = 0.010000
# Rw = 0.163814
# Pearson = 0.987273

# Target: SrFe2As2_210K.gr
# Optimized morphing parameters:
# rmax = 100.010000
# rmin = 0.000000
# rstep = 0.010000
# Rw = 0.175156
# rmax = 100.010000
# rmin = 0.000000
# rstep = 0.010000
# Rw = 0.224387
# Pearson = 0.978108

# Rw table saved to C:\Users\Spark\desktop\savefile.txt```

Reply:
If I were a user I may prefer it in a csv format for easier porting to other programs?  this will also be better on PDFitc.  No need to print anything to screen, though since you have it you may as well leave it but nest it in `if verbose`.   A summary that would be useful on screen would just be filename and Rw so I can quickly find the jump (i.e., the phase transition) without having to open a file.  What do you think?

It makes sense to me to print to screen when it is a single pair of files (in that case no way I want to open up a file from disc"